legal-api/report-templates/template-parts/business-summary/businessDetails.html

```diff
@@ -35,29 +35,51 @@
   </div>
 {% endif %}
 <div class="mt-2">
-    {% if business.legalType == 'CP' %}
-        <div class="mt-2">
-            <span class="section-sub-title">Cooperative Type: </span>
-            <span class="section-data">{{business.coopType}}</span>
+     {% if business.legalType in ['SP', 'GP'] %}
+         <div class="mt-2 section-title">Business Information</div>
+        <div class="mt-4">
+            <span class="section-sub-title">Business Name: </span>
+            <span class="section-data">{{business.legalName}}</span>
         </div>
-    {% endif %}
-
-    <div class="mt-2">
-        <span class="section-sub-title">Last Annual Report: </span>
-        <span class="section-data">{{business.last_ar_date}}</span>
-    </div>
-    {% if business.legalType == 'CP' %}
         <div>
-            <span class="section-sub-title">Last Annual General Meeting: </span>
-            <span class="section-data">{{business.last_agm_date}}</span>
+            <span class="section-sub-title">Nature of Business: </span>
+            <span class="section-data">{{business.naicsCode}} - {{business.naicsDescription}}</span>
```

Comment:
naicsCode can be empty if it has a conversion filing. right? Do we need to consider that here or any other template?

Reply:
Yes, That is a very good catch. Will add the condition